mediorum/server/serve_image.go

```diff
@@ -45,9 +47,11 @@ func (ss *MediorumServer) serveImage(c echo.Context) error {
 	}
 
 	// use cache if possible
-	if blobData, ok := ss.imageCache.Get(cacheKey); ok {
-		c.Response().Header().Set("x-image-cache-hit", "true")
-		return serveSuccessWithBytes(blobData, ss.StartedAt)
+	if !skipCache {
+		if blobData, ok := ss.imageCache.Get(cacheKey); ok && len(blobData) > 0 {
```

Comment:
is it possible for me to send some sort of range request and get the cache poisoned?